qiskit_nature/second_q/algorithms/initial_points/hf_initial_point.py

```diff
@@ -34,18 +34,29 @@ class HFInitialPoint(InitialPoint):
     :class:`~qiskit.circuit.library.initial_states.hartree_fock.HartreeFock` initial state (which
     will be prepended to the :class:`~qiskit.circuit.library.ansatzes.ucc.UCC` circuit) the all-zero
     initial point will correspond to the HF initial point.
-
-    The excitation list generated by the :class:`~qiskit.circuit.library.ansatzes.ucc.UCC` ansatz is
-    obtained to ensure that the shape of the initial point is appropriate.
     """
 
     def __init__(self) -> None:
         super().__init__()
         self._ansatz: UCC | None = None
-        self._excitation_list: list[tuple[tuple[int, ...], tuple[int, ...]]] | None = None
         self._reference_energy: float = 0.0
         self._parameters: np.ndarray | None = None
 
+    @property
+    def ansatz(self) -> UCC:
+        """The UCC ansatz.
+
+        The ``excitation_list`` and ``reps`` used by the
+        :class:`~qiskit.circuit.library.ansatzes.ucc.UCC` ansatz is obtained to ensure that the
+        shape of the initial point is appropriate.
+        """
+        return self._ansatz
+
+    @ansatz.setter
+    def ansatz(self, ansatz: UCC) -> None:
+        self._invalidate()
+        self._ansatz = ansatz
+
     @property
     def grouped_property(self) -> BaseProblem | None:
```

Comment:
I'm noticing just now that I never updated the naming of this variable in #796..
I will do that in a separate PR in order to get this one done :+1: